coherence-spring-core/src/main/java/com/oracle/coherence/spring/annotation/SubscriberGroup.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at
+ * https://oss.oracle.com/licenses/upl.
+ */
+package com.oracle.coherence.spring.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import javax.enterprise.util.AnnotationLiteral;
+
+/**
+ * A qualifier annotation used when injecting {@link com.tangosol.net.topic.Subscriber}
+ * to a {@link com.tangosol.net.topic.NamedTopic} to indicate the name of the
+ * subscriber group that the subscriber should belong to.
+ *
+ * @author Vaso Putica
+ * @since 3.0
+ */
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SubscriberGroup {
+
+	/**
+	 * The name of the subscriber group.
+	 * @return the name of the subscriber group
+	 */
+	String value();
+
+	/**
+	 * An annotation literal for the {@link SubscriberGroup} annotation.
+	 */
+	class Literal extends AnnotationLiteral<SubscriberGroup> implements SubscriberGroup {
```

Comment:
You can probably remove this whole section. I don't think we use it for anything. That will also get rid of your CheckStyle error